src/lightning/app/components/serve/auto_scaler.py

```diff
@@ -257,7 +257,7 @@ async def consumer(self):
             if batch and (is_batch_ready or is_batch_timeout):
                 self._server_status[server_url] = False
                 # find server with capacity
-                asyncio.create_task(self.send_batch(batch, server_url))
+                task = asyncio.create_task(self.send_batch(batch, server_url))
```

Comment:
You can add the comment
```
# noqa: F841
```
behind this line so flake8 will ignore the unused assignment

Reply:
In addition, a brief comment for why this is needed (e.g. as you've done in the PR description with link to python docs) would be useful too :)

Reply:
Updated :)